tools/coredumper/README

```diff
@@ -0,0 +1,28 @@
+Coredump-reader
+===============
+
+Tool for processing FW stack dumps. In verbose mode it prints DSP registers,
+and function call addresses in stack up to that which caused core dump. It
+then prints either to file or to stdin all gdb commands unwrapping those
+function call addresses to function calls in human readable format.
+
+Usage sof-coredump-reader.py [-h] [-a ARCH] [-c] [-l COLUMNCOUNT] [-v]
+				(--stdout | -o OUTFILE) [--stdin | -i INFILE]
+
+-h			show this help message and exit
+-a ARCH		determine architecture of dump file; valid archs are: LE64bit, LE32bit
+-c			set output to be colourful
+-l COLUMNCOUNT		set how many colums to group the output in
+-v			increase output verbosity
+--stdin		input is from stdin
+-i INFILE		path to sys dump bin
+--stdout		output is to stdout
+-o OUTFILE		output is to FILE
+
```

Comment:
I think it would be more readable to align all the parameter description at the same column. In my reader the parameter description for --stdin and -a ARCH are somehow different than the rest.

Reply:
Yes, sure, in the main readme file it is aligned correctly.